### ISSUE TITLE
Data pipelines public API

### ISSUE THREAD

**Issue by pvanliefland** [BLSQ/openhexa#13]
TL;DR : allow external systems / applications to easily trigger (and monitor) an OpenHexa data pipeline.

## Rationale

OpenHexa data pipelines are either scheduled to run automatically or are launched using the OpenHexa web interface.

To make OpenHexa more interoperable, we should allow third-party systems to use an API to trigger and monitor data pipelines execution.

## Approach

As we already have a GraphQL API, the plan is to test and adapt it for a third-party usage scenario.

No major refactoring is expected - most efforts will probably revolve around authentication and documentation.
